workos/events.py

```diff
@@ -0,0 +1,78 @@
+from warnings import warn
+import workos
+from workos.utils.request import (
+    RequestHelper,
+    REQUEST_METHOD_GET,
+)
+
+from workos.utils.validation import EVENTS_MODULE, validate_settings
+from workos.resources.list import WorkOSListResource
+
+RESPONSE_LIMIT = 10
+
+
+class Events(WorkOSListResource):
+    """Offers methods through the WorkOS Events service."""
+
+    @validate_settings(EVENTS_MODULE)
+    def __init__(self):
+        pass
+
+    @property
+    def request_helper(self):
+        if not getattr(self, "_request_helper", None):
+            self._request_helper = RequestHelper()
+        return self._request_helper
+
+    def get_events(
+        self,
+        events=None,
+        limit=None,
+        after=None,
+        rangeStart=None,
+        rangeEnd=None,
+    ):
+        """Gets a list of Events .
+        Args:
```

Comment:
In some places we use `Args` and in other `Kwargs`. Do you know which is correct? =)

Reply:
Ah good callout, I think `Kwargs` would be appropriate here